packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -39,8 +41,7 @@ class BpkScrollableCalendarGridList extends React.Component {
   constructor(props) {
     super(props);
 
-    this.rowRenderer = this.rowRenderer.bind(this);
-    this.renderList = this.renderList.bind(this);
+    this.ref = React.createRef();
```

Comment:
This conflicts with the `peerDependencies` of the package(this API is only available in 16.3.0 and will thus fail for `^15.0.0` which should be supported per the `peerDependencies`